relion/convert/convert_utils.py

```diff
@@ -204,6 +204,42 @@ def convertMask(img, outputPath, newPix=None, newDim=None, threshold=True):
         params += ' --threshold_above 1 --threshold_below 0'
 
     pwutils.runJob(None, 'relion_image_handler', params, env=Plugin.getEnviron())
+    return outFn
+
+def convertMaskThreshold(img, outputPath, newPix=None, newDim=None, invert=False):
```

Comment:
This is code duplication. Please use already existing convertMask function

Reply:
Why is this code duplication? The functions "convertMask" and "convertMaskThreshold" are based on different programs, relion_image_handler and relion_mask_create respectively. It is not clear for me what is duplicated and how the duplication should be handled. In convertMask everything above 1 is set to 1 and everything bellow 0 is set to zero. This is not what I need, in convertMaskThreshold any voxels is either set to 0 or to 1

Reply:
Apologies, I have read the "Convert mask to mrc format read by Relion" that you wrote, so this must be a wrong description. As for relion_mask_create, there's already a separate protocol for it in Scipion, which has all the options.

Reply:
I modified the description.  

   From my point of view convertMaskThreshold is pretty similar to convertMask. It is nor clear to me why the first function is wrong but the second is OK.  If you want I will remove the function but certainly it will make the protocol more error prone.

Reply:
the convertMask converts the input mask to mrc and checks pixel/box/values, it does not create a new mask from a volume. convertMaskThreshold does the job which is already implemented in protocol_create_mask3d.py.

Reply:
OK,  you are in charge of this plugin so if you ask me to remove convertMaskThreshold I will do it. I like the idea of keeping 
convertMaskThreshold because in this way the same object/mask that can be used as input for the  version of subtractprojectin that uses a relion protocol as input may be used for the version that  uses particles as input.

Reply:
If convertMaskThreshold function is duplicating what the specific protocol for that is doing, I agree with @azazellochg 
 to avoid duplicity and keep this functionality in the other protocol.